### PR TITLE
Misc. cleanup

### PR DESCRIPTION
**Pull request by nbhuiyan**
Signed-off-by: Nazim Uddin Bhuiyan <nazimudd@ualberta.ca>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -78,16 +78,16 @@ set(OMR_GC         ON  CACHE INTERNAL "Enable the GC")
 set(OMR_FVTEST     OFF CACHE INTERNAL "Disable OMR's internal test suite, it's incompatible with b9")
 set(OMR_WARNINGS_AS_ERRORS OFF CACHE INTERNAL "OMR doesn't compile cleanly on my laptop :p")
 
-# Compile a b9-js *.src to C++
+# Compile a b9-js *.js (formerly, *.src) to C++
```
Comment (rwy7) [inline CMakeLists.txt]: I'm not sure documenting the old .src convention is really useful here. Might as well erase any mention and declutter the project.

**Review by rwy7 — CHANGES_REQUESTED** [review]
Nice nice nice! There's just the one comment I want fixed up, then I will accept this PR. GJ fixing up the guide. I guess the next step is simplifying the module struct?

**Comment by rwy7** [comment]
@youngar are you cool with us removing the `-loop` and `-function` CLI options?

Maybe we want to replace `-function <name>` with `-function <index>` ?

File: CMakeLists.txt (hunk 1 above)
Comment (nbhuiyan) [reply CMakeLists.txt]: I don't know why I thought keeping that would be useful 😦

**Review by rwy7 — APPROVED** [review]

**Comment by rwy7** [comment]
thanks!

**Pull request merged**